modules/adot/variables.tf

```diff
@@ -36,8 +36,8 @@ variable "adot_collector_policy_arns" {
 variable "adot_config" {
   type = any
   default = {
-    drop_namespace_regex = "(cert-manager)"
-    additional_metrics   = {}
+    accepte_namespace_regex = "(cert-manager)"
```

Comment:
maybe better to change this default value to something like "(default|kube-system)"?